TIG/TIG/Feature/Home/HomeViewModel.swift

```diff
@@ -192,13 +192,14 @@ extension HomeViewModel {
         let calendar = Calendar.current
         let now = Date()
         
-        let hour = calendar.component(.hour, from: now)
-        let minute = calendar.component(.minute, from: now)
+        let currentTimeInMinutes = calendar.component(.hour, from: now) * 60 + calendar.component(.minute, from: now)
```

Comment:
```suggestion
        let hour = calendar.component(.hour, from: now)
        let minute = calendar.component(.minute, from: now)
        let currentTimeInMinutes = hour * 60 + minute
```
기존 방식처럼 hour, minute 변수를 각각 만들어서 값을 구하고 이를 currentTimeInMinutes를 계산할 때 사용하는 방식이 가독성 면에서 좀 더 좋아보입니다.

Reply:
감쟈합니다 수정했습니다